### DISCUSSION TITLE
Assign reviewers

### DISCUSSION THREAD

**Discussion by Chris-V** [BetaHuhn/repo-file-sync-action#162] [Ideas] [answered]
The actions has an option to configure "assignees" but Github only supports individual users here. It is however possible to assign teams as "reviewers".

It would be great for this action to have an additional input to configure the reviewers on the PR that it creates.

Thanks you for the awesome work.

**Comment by BetaHuhn** [comment, marked as answer]
Hi @Chris-V, makes perfect sense, we should definitely add this!

Looking at the GitHub API, it separates normal `reviewers` from `team_reviewers`, so we would have to have two inputs as well. 

Feel free to create a PR for this yourself or I will work on it sometime this week :)

**Reply by BetaHuhn** [reply]
This is included in version [v1.17.0](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.17.0) :tada:

To use the latest version, use the Action with the `v1` or `latest` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@v1
```

**Reply by Chris-V** [reply]
Woah, that was quick. Thanks a lot for the hard work!
